setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
+name = openPMD_validator
```

Comment:
as with `openPMD-viewer` we should try to use the same naming :)
https://pypi.python.org/pypi/openPMD-viewer/0.7.1